src/DynamoApplications/StartupUtils.cs

```diff
@@ -103,17 +103,6 @@ internal class CLILookUp : DynamoLookUp
             public override IEnumerable<string> GetDynamoInstallLocations()
             {
                 throw new NotImplementedException();
```

Comment:
hmm, @sm6srw does this DynamoLookup class even make sense anymore?

Reply:
CLILookUp is internal and not used anywhere. We could remove it

Reply:
UpdateManager is still used by UpdateManager that we use in our migrations. Out scope for now. But I did removed CLILookup.